docs/docs/Integrations/Composio/integrations-composio.mdx

```diff
@@ -24,33 +18,38 @@ Composio components require authentication to Composio with a Composio API key.
 
 Depending on the components you use, you may also need additional access, such as an OpenAI API key, Gmail account, or GitHub account.
 
-## Use Composio components in a flow
+## Use the Composio Tools component in a flow
+
+Connect a **Composio Tools** component to an **Agent** component to enable your agent to use Gmail functionality.
 
-1. In Langflow, create a flow.
+1. In Langflow, create a Simple Agent starter flow.
 
-2. Add an **Agent** component and a **Composio Tools** component.
+2. Add a **Composio Tools** component to the flow.
 
 3. Connect the **Agent** component's **Tools** port to the **Composio Tools** component's **Tools** port.
 
-4. In the **Composio API Key** field, enter your Composio API key.
+4. In the **Agent** component, enter your OpenAI API key or configure the **Agent** component to use a different LLM.
+5. In the **Composio API Key** field, enter your Composio API key.
```

Comment:
I believe COMPOSIO_API_KEY is also a default environment variable that Langflow can auto-detect. (see configuration-global-varaibles.mdx).

You could add that to the "Authentication for Composio components" section.